doc/devel/development_setup.rst

```diff
@@ -164,6 +164,24 @@ true for ``*.py`` files.  If you change the C-extension source (which might
 also happen if you change branches) you will have to re-run
 ``python -m pip install -ve .``
 
+Verify the Installation
+==============================
+
+Run the following command to make sure you have correctly installed matplotlib
+in editable mode. The command should be run in the development when the virtual
```

Comment:
I don't understand what "in the development" means here.

Reply:
Hi, I think we can remove "the development", just virtual env makes more sense to me now.